src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/jpa/repository/PeriodLengthRepository.kt

```diff
@@ -1,6 +1,11 @@
 package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.repository
 
+import org.springframework.data.jpa.repository.EntityGraph
 import org.springframework.data.repository.CrudRepository
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.PeriodLengthEntity
+import java.util.*
 
-interface PeriodLengthRepository : CrudRepository<PeriodLengthEntity, Int>
+interface PeriodLengthRepository : CrudRepository<PeriodLengthEntity, Int> {
+  @EntityGraph(attributePaths = ["sentenceEntity"])
```

Comment:
the  EntityGraph here forces this lookup to do a fetch join (as opposed to a lazy fetch on sentence)